content/24.vaccines-novel.md

```diff
@@ -431,59 +437,70 @@ Similar to BNT162b2, the mRNA-1273 vaccine was associated with mild-to-moderate
+Therefore, it is unsurprising that in spite of vaccination programs, infection rates and hospitalization rates climbed in early 2022 in many Western countries including the United States [@url:https://www.nytimes.com/interactive/2021/us/covid-cases.html; @{url:https://www.washingtonpost.com/health/interactive/2022/omicron-comparison-cases-deaths-hospitalizations/}], especially given that many places simultaneously began to loosen public health restrictions designed to reduce viral spread..
+
+On the side of safety, the only major concern that has been raised is a possible link between mRNA vaccination and myocarditis, especially in young men [@doi:10.15585/mmwr.mm7032e4].
+This concern began with case reports of a small number of cases of myocarditis following vaccination in several countries [@doi:10.1016/j.vaccine.2021.05.087; @doi:10.1001/jamacardio.2021.2828].
+Following these reports, the Israeli Ministry of Health began surveillance to monitor the occurrence of myocarditis [@doi:10.1056/NEJMoa2109730].
+They identified 283 cases, almost exactly half of which occurred following vaccination with Pfizer's BNT162b2.
+Close analysis of these cases determined that the vaccine did have a significant effect on the incidence of myocarditis; however, the rate of myocarditis remained low overall [@doi:10.1056/NEJMoa2109730].
+The identification of young men as a population at particular risk of this AE was supported, and the risk was found to be greater after the second dose than the first.
+However, most findings suggest that this AE does not have long-term negative effects; a 2021 meta-analysis identified 69 cases, all of which resulted in full recovery [@doi:10.7759/cureus.19240].
+Although these events are very rare, as with the possible thromboembolic AEs associated with viral-vectored DNA vaccines, these findings suggest that it may be prudent to offer a framework for decision making for patients particularly concerned about specific AEs in settings where multiple vaccines are available.
 
 ### Booster Doses
 
-Due to waning neutralizing antibodies and viral evolution, boosters have emerged as an important strategy in retaining the benefits of vaccination over time.
+Due to waning effectiveness of vaccines over time, especially in light of viral evolution, boosters have emerged as an important strategy in retaining the benefits of vaccination over time.
+Booster shots are now recommended in many places, and vaccines in the near future will likely account for multiple variants and strains of SARS-CoV-2 [@doi:10.1038/s41591-021-01230-y].
+For example, in the U.S.A., the FDA recently recommended bivalent booster doses designed to account for the Omicron VOC [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-recommends-inclusion-omicron-ba45-component-covid-19-vaccine-booster; @url:https://www.medpagetoday.com/infectiousdisease/covid19vaccine/99493; @url:https://www.fda.gov/media/159597/download].
```

Comment:
Define "bivalent?"

Reply:
```suggestion
For example, in the U.S.A., the FDA recently recommended bivalent booster doses designed to account for the Omicron VOC [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-recommends-inclusion-omicron-ba45-component-covid-19-vaccine-booster; @url:https://www.medpagetoday.com/infectiousdisease/covid19vaccine/99493; @url:https://www.fda.gov/media/159597/download].
In this case, bivalent refers to the fact that doses deliver both the original formulation and an updated vaccine designed for the Omicron subvariants circulating in summer 2022.
```

Reply:
This updated text looks good to me.  However, I can't apply the suggestion because the text is now considered outdated on GitHub.